ci/aws-iam-check-keys/find_stale_keys.py

```diff
@@ -290,14 +301,13 @@ def del_key(key_dict: dict):
     Send the key(s) to the pushgateway client to let it determine if they
     are stale
     """
-    gateway = f'{env.str("GATEWAY_HOST")}:{env.int("GATEWAY_PORT", 9091)}'
-    # del key_dict["days_since_rotation"]
+    gateway = f"{env.str('GATEWAY_HOST')}:{env.int('GATEWAY_PORT', 9091)}"
+    print(f"key dict in del is: {key_dict}")
+    del key_dict["days_since_rotation"]
```

Comment:
Is this block only what you are changing and the rest is just formatting?

Reply:
yes